recipes/eglibc/eglibc.inc

```diff
@@ -24,14 +24,17 @@ LIBTOOL_DEPENDS = ""
 
 MACHINE[unexport]       = "1"
 
-RECIPE_FLAGS:native = "toolchain_kernel_version_native"
-RECIPE_FLAGS:machine = "toolchain_kernel_version_machine"
+RECIPE_FLAGS:native = "toolchain_kernel_version_native eglibc_obsolete_rpc"
```

Comment:
We could also just add --enable-obsolete-rpc for both native and sdk always, as the minimal added size should not matter there.